bootcamp/settings.py

```diff
@@ -456,6 +457,14 @@ def get_var(name, default):
 CYBERSOURCE_REFERENCE_PREFIX = get_var("CYBERSOURCE_REFERENCE_PREFIX", None)
 
 
+# FluidReview
+FLUIDREVIEW_ACCESS_TOKEN = get_var("FLUIDREVIEW_ACCESS_TOKEN", None)
+FLUIDREVIEW_REFRESH_TOKEN = get_var("FLUIDREVIEW_REFRESH_TOKEN", None)
+FLUIDREVIEW_CLIENT_ID = get_var("FLUIDREVIEW_CLIENT_ID", None)
+FLUIDREVIEW_CLIENT_SECRET = get_var("FLUIDREVIEW_CLIENT_SECRET", None)
+FLUIDREVIEW_BASE_URL = get_var("FLUIDREVIEW_BASE_URL", None)
```

Comment:
I feel like this should exclude the `api/v2` path for consistency with our other `_BASE_URL`s, for example `BOOTCAMP_ADMISSION_BASE_URL` below. What do you think?

Reply:
sounds good